wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -177,16 +177,12 @@ October 25, 2016 (11:45 UTC)
 
 Game modes must be written as follows:
 
-- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!`
 - `osu!taiko`
 - `osu!catch`
 - `osu!mania`
 
-*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
-
-Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name.
-
-Articles such as the [Ranking Criteria](/wiki/Ranking_Criteria) may use `osu!` instead of `osu!standard`.
+Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note however that citing the osu! game mode as `osu!standard` is *not* recommended.
```

Comment:
"reccomended" doesn't really seem to be the best word to describe "not standard". Here's a few options:
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that `osu!standard` should *not* be used to refer to the osu! game mode unless speaking about it's colloquial nature.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that citing the osu! game mode as `osu!standard` is *not* prohibited unless it is speaking of its colloquial nature.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that `osu!standard` should *not* be used to refer to the osu! game mode.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that citing the osu! game mode as `osu!standard` is *not* prohibited.
```

Reply:
Why are there four suggestions here?

Reply:
I wasn't sure if the exception to the rule should be specified, so it's just different ways of saying the same thing. Only one will be applied tho, so ¯\\\_(ツ)_/¯

Reply:
All of these seem either too verbose or flip the statement around with "is not recommended" -> "is not prohibited", which to me doesn't make much sense. The previous sentence should make that clear enough.

How about
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do not otherwise cite the osu! game mode as `osu!standard`.
```
(yes, i'm adding another suggestion)

Reply:
The previous sentence does not refer to osu!standard at all, so why "otherwise"?. Doesn't really matter, though, cus this works fine, too. As long as the rule surrounding the usage of "osu!standard" is understandable and made clear, then it should be fine; just needed to get rid of "recommended".

Reply:
Hmm, right, and although a direct "do not cite" may be correct, it doesn't flow well in the paragraph. I'd go with your third suggestion in that case.

Reply:
o thanks for all the suggestions guys but i guess i'll just go with TicClick's solution in the other comment chain to delete this problematic sentence altogether :x